src/server/protocol.ts

```diff
@@ -630,66 +625,6 @@ export interface GetMoveToRefactoringFileSuggestions extends Response {
     };
 }
 
-/**
- * A set of one or more available refactoring actions, grouped under a parent refactoring.
- */
-export interface ApplicableRefactorInfo {

```

Comment:
This is an example of an interface that was copied exactly from services/types.ts. This can simply be imported instead of duplicated.